client/tests/simulator/capability.rs

```diff
@@ -0,0 +1,42 @@
+#![cfg(feature = "tpm-simulator-tests")]
```

Comment:
Unneeded, there's already a conditional compile in the parent module.

Reply:
Fixed